packages/datadog-plugin-fastify/test/index.spec.js

```diff
@@ -26,14 +28,12 @@ describe('Plugin', () => {
 
       withExports('fastify', version, ['default', 'fastify'], '>=3', getExport => {
         describe('without configuration', () => {
-          if (NODE_MAJOR <= 18 && semver.satisfies(specificVersion, '>=5')) return
-
           before(() => {
             return agent.load(['fastify', 'find-my-way', 'http'], [{}, {}, { client: false }])
           })
 
           after(() => {
-            return agent.close({ ritmReset: false })
+            return agent.close({ ritmReset: false, wipe: true })
```

Comment:
This should no longer be needed now that the tracer URL can be reconfigured and plugins can be enabled/disabled.

Reply:
If I remove it, the tests fail because it's only running the plugin constructers before the first test. I need to re-run it because the tests change the tracer config

Reply:
It should be enough to reload the agent. Did you try doing `agent.load` in a `beforeEach` instead of `before`?

Reply:
Yes, didn't fix it

Reply:
Oh, actually after refactoring the code as you suggested and letting the composite plugin take care of enabling/disabling the subscriptions based on the config option, it all works now without `wipe: true`. I'll remove it